java/semantickernel-api/src/main/java/com/microsoft/semantickernel/orchestration/ToolCallBehavior.java

```diff
@@ -38,21 +45,16 @@ static String getKey(String skillName, String functionName) {
             return String.format("%s-%s", skillName, functionName);
```

Comment:
Note difference between getNameSeparator and the separator used by getKey. Probably doesn't matter, but they should be the same. I can see a bug creeping because a function isn't found or isn't included or excluded from tools... this would be hard to track down.

Reply:
Oh, yes, I'll update the other one.